web/pdf_page_view.js

```diff
@@ -260,6 +260,10 @@ class PDFPageView {
    */
   updatePosition() {
     if (this.textLayer) {
+      let readableStream = this.pdfPage.streamTextContent({
+        normalizeWhitespace: true,
+      });
+      this.textLayer.setTextContentStream(readableStream);
```

Comment:
Considering that there were no existing `setTextContent` call here, I'm sceptical that you need to add this code here.

This would basically force *a lot* of duplicate streams being created at *every* scroll event, which seems like it would be really bad for performance. I'd assume that issue observed actually lies elsewhere, rather than in *this* particular method.